docs/source/sci_api_tutorial/dipole.rst

```diff
@@ -0,0 +1,153 @@
+
+``job_check_period``:
+    Time cycle for updating job state changes (default is 30 seconds).
+
+
+Example code
+=========================================================================================
+.. panels::
+
+    :column: col-lg-12 col-md-12 col-sm-12 col-xs-12 p-2 text-left
+
+    We will use the script to calculate the dipole moment between the ``CAE`` atom1 and ``H2`` atom2 
+
+        1. import the PDBParser class and make PDBParser instance. ``sp = PDBParser()`` and get the structure from the file provide with sp.get_structure(f"xxx.pdb")
+        2. Assign the charge and the spin for the ligand with specific name "H5J". 
+        3. Select the region which you want to do the dipole calculation with nterm and cterm capped with your desired methods
+        4. Adjust the atoms order after the capping atoms
+        5. Define the target atoms and assign to a tuple
+        6. Load the electron structure fchk file output from Gaussian16 wiht ``ElectronicStructure()`` science API
+        7. Get the result by input the params in the previous selection to ``bond_dipole``.
+        
+    .. code:: python
+
+        bond_dipole(name_ele_stru, target_bond[0], target_bond[1])
+
+**Let's put them together as a python script.**
+        
+    .. code:: python
+        
+        import glob
+        import pytest
+        import os
+        import numpy as np
+        from enzy_htp.core.clusters.accre import Accre
+        import enzy_htp.core.file_system as fs
+        from enzy_htp.structure.structure_region import create_region_from_selection_pattern
+        from enzy_htp.electronic_structure import ElectronicStructure
+        from enzy_htp.analysis import bond_dipole
+        from enzy_htp import interface
+        from enzy_htp import PDBParser
+
+        DATA_DIR = f"{os.path.dirname(os.path.abspath(__file__))}/data/"
+        STRU_DATA_DIR = f"{os.path.dirname(os.path.abspath(__file__))}/../test_data/diversed_stru/"
+        WORK_DIR = f"{os.path.dirname(os.path.abspath(__file__))}/work_dir/"
+        sp = PDBParser()
+
+        ke_stru = sp.get_structure(f"{DATA_DIR}KE_mutant_101_254_frame_0.pdb")
+        ke_stru.assign_ncaa_chargespin({"H5J" : (0,1)})
+        test_region = create_region_from_selection_pattern( ke_stru, "resi 101+254", nterm_cap = "H", cterm_cap = "H",)
+        
+        cap_h_1 = test_region.atoms.pop(-2)
+        test_region.atoms.insert(1, cap_h_1)
+        cap_h_2 = test_region.atoms.pop(-1)
+        test_region.atoms.insert(15, cap_h_2)
+        target_bond = (ke_stru.ligands[0].find_atom_name("CAE"), ke_stru.ligands[0].find_atom_name("H2")) 
+        
+        ke_ele_stru = ElectronicStructure(energy_0 = 0.0, geometry = test_region, mo = f"{DATA_DIR}KE_mutant_101_254_frame_0.fchk", mo_parser = None,source ="gaussian16") 
+        result = bond_dipole(test_ele_stru, target_bond[0], target_bond[1],work_dir=WORK_DIR)
```

Comment:
You should highlight that this is where the API is used. You can do it by formatting the comment like
```
# ======= API call ==========
...
```